package.json

```diff
@@ -26,22 +26,53 @@
 		"xo"
 	],
 	"dependencies": {
+		"@atom/babel7-transpiler": "^1.0.0-1",
+		"@babel/preset-env": "^7.12.1",
+		"@babel/preset-typescript": "^7.12.1",
 		"atom-linter": "^10.0.0",
 		"atom-package-deps": "^5.0.0",
 		"eslint-rule-documentation": "^1.0.0",
 		"load-json-file": "^5.1.0",
 		"p-props": "^1.0.0",
-		"pkg-dir": "^3.0.0",
+		"pkg-dir": "^5.0.0",
 		"resolve-from": "^4.0.0",
-		"xo": "^0.24.0"
+		"xo": "^0.34.0"
 	},
 	"devDependencies": {
-		"@babel/register": "^7.0.0",
-		"ava": "^1.1.0",
+		"@types/atom": "^1.40.5",
+		"@types/jasmine": "^3.6.0",
+		"@types/text-buffer": "^13.0.4",
+		"@types/tmp": "^0.2.0",
```

Comment:
I only converted `lib/get-package-path.ts` to start with. These other `@types` packages are used elsewhere.